app/models/services/pubmed/resource.rb

```diff
@@ -27,6 +27,7 @@ def mapper
         doi:                lambda {|data| data.css('ArticleId[IdType=doi]').text },
         pubmed_id:          lambda {|data| data.css('PMID').text },
         abstract:           lambda {|data| data.css('AbstractText').map(&:text).join("\n\n") },
+        abstract_editable:  lambda {|data| false if data.css('AbstractText').present? },
```

Comment:
I personally try to avoid negatives, so I'd rather `true if data.css('AbstractText').blank?`